projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/product-list.ts

```diff
@@ -0,0 +1,56 @@
+import { checkAllElements, TabElement } from '../tabbing-order';
+import { formats } from '../../../sample-data/viewports';
+
+export function productListTabbingOrderDesktop(
+  config: TabElement[],
+  grid: boolean = false
+) {
+  cy.visit('/Brands/all/c/brands');
```

Comment:
we can export this url, similarly to `testProductUrl`

Reply:
Done